response_operations_ui/common/respondent_utils.py

```diff
@@ -29,7 +29,7 @@ def filter_respondents(respondents):
     for respondent in respondents:
         try:
             filtered_respondents.append({
-                'href': '/respondent-details/' + str(respondent['id']),
+                'href': '/respondents' + '/respondent-details/' + str(respondent['id']),
```

Comment:
I'd use a formatted string.
```suggestion
                'href': f'/respondents/respondent-details/{str(respondent['id'])}',
```